test/videoframeprocessor/DefaultVideoTransformDevice.test.ts

```diff
@@ -73,7 +73,7 @@ describe('DefaultVideoTransformDevice', () => {
     it('returns null if device is not started', async () => {
       const processor = new NoOpVideoFrameProcessor();
       const device = new DefaultVideoTransformDevice(logger, 'test', [processor]);
-      expect(await device.outputMediaStream).to.be.null;
+      expect(await device.outputMediaStream).to.be.undefined;
     });
```

Comment:
This isn't right. The description of the test is no longer accurate, but more importantly it means `NoOpVideoFrameProcessor` is not returning a stream when the type signature demands that it should.

Further to our earlier conversation, this is exactly the kind of thing that strict typing *would* catch, because` outputMediaStream` is not allowed to be `undefined`.